src/assets/apple-ios.svg

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
FYI, I've used [this validator](https://validator.nu/) in the past to help me validate and manually minify our SVGs. There's usually some metadata that can be stripped out.